core/lib/db/queries.ts

```diff
@@ -57,6 +57,7 @@ export const getStageMembers = cache((stageId: StagesId) => {
 			.innerJoin("users", "users.id", "stage_memberships.userId")
 			.select(SAFE_USER_SELECT)
 			.select("stage_memberships.role")
+			.orderBy("users.id")
```

Comment:
i don't really care, but is this necessary? kind of a weird thing to order on imo

Reply:
Yeah this was the reason: new members were being added to the panel at a random point. But ordering by id isn't much better, so I changed it to createdAt